test/UnitPerf.cpp

```diff
@@ -91,14 +91,15 @@ void UnitPerf::invokeWSDTest()
 {
     TST_LOG("startup: " << _timer->elapsedTime().count() << "us\n");
     _timer->restart();
-
-    testPerf("writer", "odt", "/../traces/perf-writer.txt");
-
-    testPerf("calc", "ods", "/../traces/perf-calc.txt");
-
-    testPerf("impress", "odp", "/../traces/perf-impress.txt");
-
-    testPerf("draw", "odg", "/../traces/perf-draw.txt");
+    std::string opt;
+    opt  = "writer";
+    testPerf( opt, "odt", "/../traces/perf-writer.txt");
```

Comment:
I'm not sure how this relates to `const` or `std::move()`. Perhaps you can explain why you thought this file needed this change, given that we have thousands of literal strings. In fact, why move "writer" from the `testPerf()` call and leave "odt", or  "/../traces/perf-writer.txt" ?